dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -28,26 +28,19 @@ function DatePickerWidget({
       : "To date must be less than from date";
 
   const onFromChange = (_str, date) => {
-    setFromDate(new Date(date));
+    setFromDate(date);
```

Comment:
I think it is fine(and maybe a good approach also),since I think we don’t want user to use filter functionality if the user types a wrong date or an ill formatted one,and in that case if fromDate is set to undefined(or an empty object),we would be able to achieve our goal and force user to enter a correct date